<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Serveur1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Serveur2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Serveur3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QMdiArea" name="mdiArea">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="background">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QMdiArea" name="mdiArea_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="background">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QMdiArea" name="mdiArea_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="background">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
    </brush>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
